README.md

```diff
@@ -414,19 +414,17 @@ set PATH=C:\STL\out\x64\out\bin\amd64;%PATH%
 You don't need any extra steps to run with test code and the code in STL headers instrumented with [ASan][].
 The test matrices include both ASan and non-ASan configurations.

```

Comment:
Maybe add "unless `-Dnotags=ASAN` is specified" somewhere here, while we're at it.

Reply:
```suggestion
The test matrices include both ASan and non-ASan configurations if you don't pass `-Dtags=ASAN` or `-Dnotags=ASAN` to exclude one or the other.
```
(This probably needs wrapping.)